src/components/partials/item-info/item-info.js

```diff
@@ -30,12 +32,14 @@ const ItemInfo = ({
         <div className="flex-column">
           <div className="flex-row align-center">
             <div className="item-info-name">
+            <Link className="item-info-link " to={itemInfoLink}>
```

Comment:
this applies the link only on the name
Let me know if it should cover the entire card instead

Reply:
Link on name is just fine :D